packages/create-cloudflare/src/helpers/interactive.ts

```diff
@@ -245,22 +253,35 @@ export const spinner = () => {
 			startMsg = `${currentMsg} ${dim(helpText)}`;
 
 			let index = 0;
-			let dots = 1;
 
 			clearLoop();
 			loop = setInterval(() => {
-				const frame = frames[(index = ++index % frames.length)];
-				dots = ++dots % maxDots;
-				logUpdate(`${color(frame)} ${currentMsg} ${".".repeat(dots)}`);
+				index++;
+				const spinnerFrame = spinnerFrames[index % spinnerFrames.length];
+				const ellipsisFrame = ellipsisFrames[index % ellipsisFrames.length];
+
+				if (msg) {
+					if (!process.env.VITEST) {
+						logUpdate(`${color(spinnerFrame)} ${currentMsg} ${ellipsisFrame}`);
+					} else {
+						updateStatus(msg);
+					}
+				}
 			}, frameRate);
 		},
 		update(msg: string) {
 			currentMsg = msg;
 		},
-		stop: (msg: string) => {
+		stop: (msg?: string) => {
 			// Write the final message and clear the loop
 			logUpdate.clear();
-			logUpdate(`${leftT} ${startMsg}\n${grayBar} ${msg}`);
+			if (msg) {
+				if (!process.env.VITEST) {
```

Comment:
How about we just bail out of spinners altogether when running tests?
And then just have the check in one place in the code?

Reply:
Ideally the spinner would just be mocked but I didn't know whether it was worth touching the tests while they were failing. But yeah I can move this to the start of the function and return a stub.

What's your opinion on falling back to updateStatus calls so that there is _something_ in the test output?